Sources/tart/Root.swift

```diff
@@ -96,6 +96,10 @@ struct Root: AsyncParsableCommand {
         try command.run()
       }
     } catch {
+      if exitCode(for: error).rawValue == 0 {
```

Comment:
Should we add some comment for future reference, as this looks a bit magical?

Reply:
Would've added if not the auto-merge. Now the context will be in git history 😅😪